v4-client-js/examples/composite_example.ts

```diff
@@ -1,7 +1,9 @@
+import { Order_TimeInForce } from '@dydxprotocol/v4-proto/src/codegen/dydxprotocol/clob/order';
```

Comment:
Is this the correct `Order_TIF` we want to be using as a a parameter? Note that I took out everything related to OrderExecution too

Reply:
Added a translation layer in example to translate `orderExecution` string to Order_TimeInForce